src/components/Buttons/Button.js

```diff
@@ -238,10 +238,14 @@ const Button = props => {
       props.onMouseDown(e, props)
     },
     ...props.elementAttributes,
+    ...props.customElementProps,
   }
   if (props.href) {
     finalProps.href = props.href
   }
+  if (props.customElement) {
```

Comment:
this is very opinionated and incorrect for some use cases

for eg: `<a href="xxx" role="button" />` doesn't make sense

i think we should remove this and let users pass it as necessary in `ElementAttributes`.